frontend/packages/console-shared/src/components/kebab/KebabItems.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { KEY_CODES, Tooltip } from '@patternfly/react-core';
+import { useAccessReview } from '@console/internal/components/utils';
+import { impersonateStateToProps } from '@console/internal/reducers/ui';
+import { Action } from '@console/dynamic-plugin-sdk';
+
+export type KebabItemProps = {
+  option: Action;
+  onClick: (event: React.MouseEvent<{}>, action: Action) => void;
+  autoFocus?: boolean;
+  onEscape?: () => void;
+};
+
+// eslint-disable-next-line no-underscore-dangle
+const KebabItem_: React.FC<KebabItemProps & { isAllowed: boolean }> = ({
```

Comment:
Rename to avoid eslint disable
```suggestion
const KebabItemButton: React.FC<KebabItemProps & { isAllowed: boolean }> = ({
```

Reply:
Done.